code/unit_tests/icon_tests.dm

```diff
@@ -141,3 +145,23 @@
 	else
 		pass("All item modifiers have valid icon states.")
 	return 1
+
+
+/datum/unit_test/icon_test/areas_shall_have_valid_icon_states
+	name = "Areas shall have valid icon states"
+
+/datum/unit_test/icon_test/areas_shall_have_valid_icon_states/start_test()
+	var/list/missing_states = list()
+	var/list/valid_states_by_icon = list()
```

Comment:
Might just be missing something, but the valid states list doesn't seem to be populated anywhere?

Reply:
See the last part of https://github.com/NebulaSS13/Nebula/pull/1981/files#diff-ce5433b1d92afe33c5cd0a21a226a3e483052b3de88b47f3b27fd98c4d9954bbR158

Reply:
Ah I see. Could use `check_state_in_icon()` instead.